LibsNoSamples.sln

```diff
@@ -1,108 +1,115 @@
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Identity.Client", "msal\src\Microsoft.Identity.Client\Microsoft.Identity.Client.csproj", "{8B57084E-12B8-4499-B7DD-D8DD882BDA98}"
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.IdentityModel.Clients.ActiveDirectory", "adal\src\Microsoft.IdentityModel.Clients.ActiveDirectory\Microsoft.IdentityModel.Clients.ActiveDirectory.csproj", "{A71924EE-1172-4725-9E2C-6E5A342D1620}"
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Test.ADAL.NET.Common", "adal\tests\Test.ADAL.NET.Common\Test.ADAL.NET.Common.csproj", "{412F8F05-2694-4D2E-8FE0-A821721CF682}"
-EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tests", "tests", "{5D231979-0A13-456D-B9F9-CED41F935FD9}"
-EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "build", "build", "{48CF4D7C-5E2E-4C4A-BDD2-8F2A06AAE3F6}"
-	ProjectSection(SolutionItems) = preProject
-		build\SolutionWideAnalyzerConfig.ruleset = build\SolutionWideAnalyzerConfig.ruleset
-	EndProjectSection
-EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "adal", "adal", "{DC4A6EAD-07D8-480A-A111-F2F8C359735B}"
-EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "msal", "msal", "{459B71BF-176E-4E1D-B77A-D21862DB115E}"
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Test.MSAL.NET.Unit.netcore", "msal\tests\Test.MSAL.NET.Unit.netcore\Test.MSAL.NET.Unit.netcore.csproj", "{AF0E7BEC-90A3-46B4-AEDC-4162D7873AE8}"
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Test.MSAL.NET.Unit.net45", "msal\tests\Test.MSAL.NET.Unit.net45\Test.MSAL.NET.Unit.net45.csproj", "{DA8E7933-0A56-4668-9786-ECECA3D16944}"
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Test.MSAL.NET.Common", "msal\tests\Test.MSAL.NET.Common\Test.MSAL.NET.Common.csproj", "{30287EC8-BD85-41CC-BDCB-9DBF9EFBCC98}"
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Test.ADAL.NET.Unit.net45", "adal\tests\Test.ADAL.NET.Unit.net45\Test.ADAL.NET.Unit.net45.csproj", "{91E9E564-8A44-457B-A97C-E76C9F6275D6}"
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Test.ADAL.NET.Unit.netcore", "adal\tests\Test.ADAL.NET.Unit.netcore\Test.ADAL.NET.Unit.netcore.csproj", "{FCDDA8D3-F299-426B-9CAD-0824D941ED53}"
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Test.Microsoft.Identity.LabInfrastructure", "core\tests\Test.Microsoft.Identity.LabInfrastructure\Test.Microsoft.Identity.LabInfrastructure.csproj", "{E22918A8-8FE6-47D2-B65E-34436918C5D7}"
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Test.Microsoft.Identity.SideBySide", "core\tests\Test.Microsoft.Identity.SideBySide\Test.Microsoft.Identity.SideBySide.csproj", "{D3033A40-F2C2-4495-BC9F-C3C6D497D1CC}"
-EndProject
-Global
```

Comment:
I can't see through these changes. diff for me claims everything has changed, which ideally it shouldn't as just a project has been added. Are you also seeing the file update completely?

Reply:
yes, i see that too.  probably a line ending issue, but also sln files are notoriously bad at diffing.  i'll see if i can regen the sln file add and fix the diff.

Reply:
Added it again.  local diff showed just the line changes, github diff shows everything different.  from local diff, i can confirm it's only adding the msal.benchmarks file.

![image](https://user-images.githubusercontent.com/7103178/48664710-abca0d00-ea57-11e8-9883-16b94e37f8cb.png)